common/test/acceptance/tests/discussion/test_discussion.py

```diff
@@ -236,6 +236,22 @@ def test_mathjax_rendering(self):
         self.assertTrue(self.thread_page.is_mathjax_preview_available())
         self.assertTrue(self.thread_page.is_mathjax_rendered())
 
+    def test_markdown_reference_link(self):
+        sample_link = "http://example.com/colon:test"
```

Comment:
I prefer that tests have documentation explaining what they are testing. Our best practice is to use BDD-style test descriptions, but at the least, add a documentation block.